index.html

```diff
@@ -0,0 +1,58 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-u2Qo5c5wL2fZ5m0z/rqZjLmZ9+vHjE/AqY0rU/QdugnfeE0k8zO6rU2qzUITjtWc6i1Kc2nh/gPmKv5fK/xJwg==" crossorigin="anonymous" >
+
+  </head>
+  <body>
+    <div class="portfolio">
+        <header class="details">
+            <div class="name">
+                <span class="inside-name">Kelvin</span>
+            </div>
+            <nav>
+            <ul class="nav-list">
+                   <li><a href="#" class="item-home">Portfolio</a></li>
+                    <li><a href="#" class="item-about">About</a></li>
+                    <li><a href ="#"class="item-contact">Contact</a></li>
+                    <li><a href="#"class="item-email">
+                        <img src="images/Disabled.png"alt="Disabled">
```

Comment:
- [ ] Since you are building this webpage with a mobile-first approach, kindly remove these links and add a hamburger, as shown in the Figma design instead.

## Your Design:
![image](https://user-images.githubusercontent.com/92215878/213514009-e16c4dff-2327-4ded-8e64-5610fd3d84b1.png)

## Figma Design:
![image](https://user-images.githubusercontent.com/92215878/213514105-b228cfe9-2cae-4da9-9baa-79dd7de0e51c.png)